cloud-controller-manager/do/cloud.go

```diff
@@ -96,6 +97,16 @@ func init() {
 }
 
 func (c *cloud) Initialize(clientBuilder controller.ControllerClientBuilder) {
+	clientset := clientBuilder.ClientOrDie("do-shared-informers")
+	sharedInformer := informers.NewSharedInformerFactory(clientset, 0)
+
+	tags := NewTagsController(sharedInformer.Core().V1().Nodes(), clientset, c.client)
+
+	// TODO: pass in stopCh once supported upstream
+	// see https://github.com/kubernetes/kubernetes/pull/70038 for more details
+	sharedInformer.Start(nil)
+	sharedInformer.WaitForCacheSync(nil)
+	go tags.Run()
```

Comment:
This is not something I liked. Maybe we should pass a `ctx` and cancell it ourself? Do we have to wait for `stopCh` support ?

Reply:
`stopCh` support PR is already merged so it will definitely be supported in 1.13. Do you think it's still worth passing a ctx and cancelling it ourselves?

Reply:
@andrewsykim it's ok then. Could you create an additional issue so we can track this down later when 1.13 is released.